infer/src/base/CommandLineOption.mli

```diff
@@ -174,17 +175,18 @@ val mk_command_doc :
     command_exe] records information about a command that is used to create its man page. A lot of
     the concepts are taken from man-pages(7).
 
-    - [command_exe] is the name of the command, preferably an executable that selects the command
-    - [title] will be the title of the manual
-    - [section] will be the section of the manual (the number 7 in man-pages(7))
-    - [version] is the version string of the command
-    - [date] is the date of the last modification of the manual
-    - [short_description] is a one-line description of the command
-    - [options] can be either [`Replace blocks], which populates the OPTIONS section with [blocks],
+    {ul
+     {- [command_exe] is the name of the command, preferably an executable that selects the command}
+     {- [title] will be the title of the manual}
+     {- [section] will be the section of the manual (the number 7 in man-pages(7))}
+     {- [version] is the version string of the command}
+     {- [date] is the date of the last modification of the manual}
+     {- [short_description] is a one-line description of the command}
+     {- [options] can be either [`Replace blocks], which populates the OPTIONS section with [blocks],
     or [`Prepend blocks], in which case the options from the command are used, prepended by
-    [blocks]. If unspecified it defaults to [`Prepend []].
-    - All the other [section_name] options correspond to the contents of the section [section_name].
-    Some are mandatory and some are not.
+    [blocks]. If unspecified it defaults to [`Prepend []].}
+    {- All the other [section_name] options correspond to the contents of the section [section_name].
+    Some are mandatory and some are not.}}
```

Comment:
I'm not sure why the heavy syntax is needed here. Is there a different edit that can be used to keep the lighter syntax?